pkg/helpers/config/userdata.yaml

```diff
@@ -1,13 +1,13 @@
 #cloud-config
-repo_update: true
-package_update: true
-packages:
-  - docker 
 runcmd:
-  - sudo service docker start
+  # taking care manually of the packages to be able to make it non-blocking
+  - sudo yum -y update || echo "didn't update package"
```

Comment:
do we need `update` and `install docker` given it'll be already done on AMI?

Reply:
because we give the option to provide different AMIs, we likely still have to do this. For e.g when Cu provides an ami without docker installed.

Reply:
maybe we could skip it, when using our AMIs